plugins/uefi/fu-plugin-uefi.c

```diff
@@ -102,11 +102,17 @@ fu_plugin_add_security_attrs (FuPlugin *plugin, FuSecurityAttrs *attrs)
 	/* create attr */
 	attr = fwupd_security_attr_new (FWUPD_SECURITY_ATTR_ID_UEFI_SECUREBOOT);
 	fwupd_security_attr_set_plugin (attr, fu_plugin_get_name (plugin));
-	fwupd_security_attr_add_flag (attr, FWUPD_SECURITY_ATTR_FLAG_RUNTIME_ISSUE);
 	fu_security_attrs_append (attrs, attr);
 
+	/* SB not available */
+	if (!fu_efivar_exists (FU_EFIVAR_GUID_EFI_GLOBAL, "SecureBoot")) {
```

Comment:
What do you think about instead making `fu_efivar_secure_boot_enabled` to take an argument of `GError` and then on the FALSE scenarios (missing variable, or disabled) explicitly put the reason in the error?

Reply:
It's a good idea; do we care about libfwupdplugin API now? I can't remember what policy is on breaking that.

Reply:
ABI yes, API, not so much.  I guess this would break ABI, so maybe it needs to be a new `_full` function then

Reply:
I wonder if we should expand DB things out into it's own class, so that we can easily get the PK, status, etc.

Reply:
Makes sense to me

Reply:
> needs to be a new _full function

I did this, a whole new object just seemed like a waste for two bits of data.